js/src/Mark.ts

```diff
@@ -197,13 +197,15 @@ export abstract class Mark extends widgets.WidgetView {
         this.d3el.style("display", visible ? "inline" : "none");
     }
 
-    get_colors(index) {
+    //how do I know if smth is a color attr or a data attr, this changes the type
```

Comment:
Maybe remove this comment

Reply:
Yeah, I'll remove all of them, it was just me writing along the way so I could make questions in the review :)